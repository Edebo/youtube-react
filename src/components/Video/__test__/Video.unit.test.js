import React from "react";
import { shallow } from "enzyme";
import { Video } from "../Video";

describe("testing video component", () => {
  test("rendering video with props id", () => {
    const wrapper = shallow(<Video id='HAuXJVI_bUs' />);
    expect(wrapper).toMatchSnapshot();
  });

  test("renders null if id in video component not specified ", () => {
    const wrapper = shallow(<Video />);

    expect(wrapper).toMatchSnapshot();
  });
});
