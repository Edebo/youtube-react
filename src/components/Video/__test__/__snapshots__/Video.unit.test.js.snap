// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing video component rendering video with props id 1`] = `
<div
  className="video-container"
>
  <div
    className="video"
  >
    <iframe
      allow="autoplay; encrypted-media"
      allowFullScreen={true}
      className="video-player"
      frameBorder="0"
      height="315"
      src="https://www.youtube.com/embed/HAuXJVI_bUs?autoplay=1"
      title="video"
      width="560"
    />
  </div>
</div>
`;

exports[`testing video component renders null if id in video component not specified  1`] = `""`;
